src/main/java/ch/njol/skript/lang/SkriptParser.java

```diff
@@ -249,6 +250,14 @@ public boolean hasTag(String tag) {
 									continue;
 								}
 							}
+							if (element instanceof ExperimentRestrictedSyntax experimentRestrictedSyntax) {
+								Feature feature = experimentRestrictedSyntax.requiredExperiment();
+								if (!getParser().hasExperiment(feature)) {
+									Skript.error("This syntax element is experimental. To enable this experiment, add "
+										+ "'using " + feature.codeName() + "' in this script outside all other code.");
+									continue;
```

Comment:
If I was a person unfamiliar with code terms, I wouldn't be able to understand "outside of all other code", I believe something along the lines of "at the top of your file" or something along the lines would be nicer.